frontend/__mocks__/hooks/api/aliases.ts

```diff
@@ -111,9 +111,15 @@ function getReturnValue(
       mutate: jest.fn(),
       data: customAliasData,
     },
-    create: callbacks?.creater || jest.fn(),
-    update: callbacks?.updater || jest.fn(),
-    delete: callbacks?.deleter || jest.fn(),
+    create:
+      callbacks?.creater ||
+      jest.fn(() => Promise.resolve({ ok: true } as unknown as Response)),
+    update:
+      callbacks?.updater ||
+      jest.fn(() => Promise.resolve({ ok: true } as unknown as Response)),
+    delete:
+      callbacks?.deleter ||
+      jest.fn(() => Promise.resolve({ ok: true } as unknown as Response)),
```

Comment:
Ah yes, I actually intended to do that and then forgot :P There were tests that verified that aliases were shown, but none that actually tried to modify aliases - so these mock functions were never called. However, if I want to verify that the notification is actually sent after an alias is created or deleted, the promises returned from these functions need to actually resolve - so that the code following it, i.e. that sends the event, is executed.